go/vt/servenv/servenv.go

```diff
@@ -98,6 +99,8 @@ func RegisterFlags() {
 
 		// pid_file.go
 		fs.StringVar(&pidFile, "pid_file", pidFile, "If set, the process will write its pid to the named file, and delete it on graceful shutdown.")
+		// Logging
+		fs.BoolVar(&usePSLogger, "structure_logging", usePSLogger, "whether to use structure logging (Planet Scale Log) logger or the original (glog) logger")
```

Comment:
Adding a flag requires changing the expected outputs for the test `TestHelpOutput`

Reply:
Also this flag should use `-` instead of `_`. So the flag should be called `structure-logging` instead

Reply:
a minor nit on the naming: this should be called _structured_ logging, not _structure_ logging